core/storage/base_model/gae_models.py

```diff
@@ -429,6 +430,7 @@ def _compute_snapshot(self):
         return self.to_dict(exclude=['created_on', 'last_updated'])
 
     def _reconstitute(self, snapshot_dict):
+        """Populates the model instance with the snapshot."""
```

Comment:
Hi @vibhor98, don't we require `Args` for `snapshot_dict` here?

Reply:
Hi @apb7 I have added it for now; will remove if turns out to be unnecesary. I too am skeptical about the way I wrote the return type. PTAL. Thanks!

Reply:
It's optional in test files. Else, we need to write them. @YashJipkate, thanks for adding them here!